src/ace/managers/ptplayer.c

```diff
@@ -2514,7 +2517,7 @@ static void set_sampleoffset(
 	else {
 		pChannelData->n_sampleoffset = ubArg;
 	}
-	UWORD uwLength = ubArg << 7;
+	UWORD uwLength = ubArg << 6;
```

Comment:
I'd reword `<< 6` to something like `* (256 / sizeof(UWORD))` to indicate the intent - ubArg is length in 256s of bytes, but we need the length in words, I assume. Perhaps also add a comment if that kind of notation is not clear enough.

Reply:
```suggestion
	// Offset is in 256s of bytes, length is in words
	UWORD uwLength = ubArg * (256 / sizeof(UWORD));
```